packages/editor/src/lib/primitives/geometry/geometry-constants.ts

```diff
@@ -2,5 +2,6 @@ const SPACING = 20
 const MIN_COUNT = 8
 
 export function getVerticesCountForLength(length: number, spacing = SPACING) {
-	return Math.max(MIN_COUNT, Math.ceil(length / spacing))
+	const num = Math.max(MIN_COUNT, Math.ceil(length / spacing))
+	return num + (4 - (num % 4))
```

Comment:
just perusing this PR to see what the vibes are :)
why this '4' change, for example? we sure it's vibin' the right things? heh